assets/src/editor/events/tickets/editor-ticket/editor-ticket-entities-list.js

```diff
@@ -91,6 +92,10 @@ const EditorTicketEntitiesList = ( { ...otherProps } ) => {
 		ticketsSortedBy,
 		Array.isArray( tickets ) ? tickets.length : 0,
 	] );
+	const {
+		loadingTicketForm,
+		setLoadingTicketForm,
+	} = useTicketFormLoadingState();
```

Comment:
Why not `useState()`?

Reply:
good question!! :laughing:

I think maybe I initially thought that more was going to need to happen in that hook and then I just didn't realize how idiotic the final code was :(